lib/Backend/NativeCodeData.h

```diff
@@ -42,6 +42,7 @@ class NativeCodeData
 
     struct DataChunkNoFixup
     {
+        unsigned int len; // only used for memory management purposes.
```

Comment:
Review: This is one of the things that I'm looking to perf and functionality test.